php/php.editor/src/org/netbeans/modules/php/editor/parser/astnodes/Scalar.java

```diff
@@ -29,7 +29,7 @@ public class Scalar extends Expression {
 
     public enum Type {
         INT, // 'int'
-        REAL, // 'real'
+        FLOAT, // 'float'
```

Comment:
I don't think we should change this because this is an API.

Reply:
Right, we cannot do this. If you really want to do that, the steps should be like:
- in release X
   - deprecate the `REAL` type + document all the details (why deprecated and when it will be removed)
   - add new `FLOAT` type
   - change the code under our control to start using the `FLOAT` type
- in release X + Y (Y is quite a big number, we cannot do it in the near future, we need to give time to our API consumers - plugins - to change their code):
   - remove the `REAL` type

Reply:
@tmysik Thank you very much for the explanation. 

Updated the PR:
- removed the `real` type removal in the public api.
- increased the minor version of the module.

I hope I did everything correctly.